fmn/lib/models.py

```diff
@@ -70,28 +81,25 @@ def init(db_url, alembic_ini=None, debug=False, create=False):
     """ Create the tables in the database using the information from the
     url obtained.
 
-    :arg db_url, URL used to connect to the database. The URL contains
-        information with regards to the database engine, the host to
-        connect to, the user and password and the database name.
-          ie: <engine>://<user>:<password>@<host>/<dbname>
-    :kwarg alembic_ini, path to the alembic ini file. This is necessary
-        to be able to use alembic correctly, but not for the unit-tests.
-    :kwarg debug, a boolean specifying wether we should have the verbose
-        output of sqlalchemy or not.
-    :return a session that can be used to query the database.
-
+    .. deprecated:: 1.2.0
+       Use the session created in this module and the fmn-createdb script instead
+
+    Args:
+        db_url (str): URL used to connect to the database. The URL contains
+            information with regards to the database engine, the host to
+            connect to, the user and password and the database name.
+            ie: <engine>://<user>:<password>@<host>/<dbname>
+        alembic_ini (str): path to the alembic ini file. This is necessary
+            to be able to use alembic correctly, but not for the unit-tests.
+        debug (bool): a boolean specifying wether we should have the verbose
+            output of sqlalchemy or not.
+
+    Returns:
+        scopedsession: An SQLAlchemy scoped session.
```

Comment:
This docblock change seems unrelated to the purpose of the rest of this commit. I recommend splitting it into a separate commit.